### PR TITLE
docs(astro/uxds): fix progress typos

### PR DESCRIPTION
**Pull request by amalyah**
## Brief Description

Fixed three typos in the Appearance & Behavior Section. seeconds > seconds. takees > takes. th > the.

## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/astro-uxds/_content/components/progress.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ When operations take one second or longer to complete, add a progress feedback e
 
 ## Appearance and Behavior
 
-An indeterminate progress indicators is displayed as a continuously spinning circle with no strict percentage or timeframe labeled. This type of indicator is usually used when a process takees between 1 to 10 seeconds unless the duration is known.
+An indeterminate progress indicators is displayed as a continuously spinning circle with no strict percentage or timeframe labeled. This type of indicator is usually used when a process takees between 1 to 10 seconds unless the duration is known.
```
Comment (micahjones13) [inline packages/astro-uxds/_content/components/progress.md:30]: Another one here I think. `takees`

Comment (micahjones13) [reply packages/astro-uxds/_content/components/progress.md:30]: Someone was really 'e' key happy

**Review by micahjones13 — CHANGES_REQUESTED** [review]

File: packages/astro-uxds/_content/components/progress.md (hunk 1 above)
Comment (amalyah) [reply packages/astro-uxds/_content/components/progress.md]: Fixed! Good eye.

**Comment by amalyah** [comment]
Looks like the commit-lint test failed due to an old commit message "docs(astro-uxds): Updated clock.md in astrouxds (#219)" with a capital letter. @micahjones13 is this something you have the power to work around or should I try to recreate my PR for these fixes?

**Review by micahjones13 — APPROVED** [review]

**Pull request merged**